### PR TITLE
Tickets/dm 35022: Add new repo of dc2 config templates

### PR DESCRIPTION
**Pull request by brendal4**

### CODE REVIEW

File: .github/workflows/yamllint.yaml [hunk 1]
```diff
@@ -0,0 +1,11 @@
+name: yamllint
+
+on:
+  push:
+    branches:
+      - main
+  pull_request:
+
```
Comment (taranu) [inline .github/workflows/yamllint.yaml:8]: Please add a configuration for 110 character line lengths here. See [the github action docs](https://github.com/marketplace/actions/yaml-lint) under `Config data examples:` (multi-line). You'll want a rule like this:

```
config_data: |
  extends: default
  rules:
    line-length:
      max: 110
    document-start:
      present: false
```

File: .github/workflows/yamllint.yaml [hunk 2]
```diff
@@ -0,0 +1,11 @@
+name: yamllint
+
```
Comment (taranu) [inline .github/workflows/yamllint.yaml:2]: See below on disabling the `document-start` check.

File: weekly_templates/dc2_pipeline_bps_step1.yaml [hunk 3]
```diff
@@ -0,0 +1,42 @@
+pipelineYaml: "$DRP_PIPE_DIR/pipelines/LSSTCam-imSim/DRP-test-med-1.yaml#step1"
+# qgraphFile: /scratch/brendal4/bps-gen3-dc2/submit/2.2i/runs/test-med-1/w_2021_40/DM-32024/20211005T175202Z/2.2i_runs_test-med-1_w_2021_40_DM-32024_20211005T175202Z.qgraph
```
Comment (taranu) [inline weekly_templates/dc2_pipeline_bps_step1.yaml]: Please remove any comments that are no longer relevant, and trim the ones that are too long (split any paths on multiple lines if needed).

File: weekly_templates/dc2_pipeline_bps_step1.yaml [hunk 4]
```diff
@@ -0,0 +1,42 @@
...
+    requestMemory: 8196
+  matchCatalogsTract:
+    requestMemory: 32000
+  wPerp:
+    requestMemory: 32000
+  TE1:
+    requestMemory: 16384
+  TE2:
+    requestMemory: 16384
+
```
Comment (taranu) [inline weekly_templates/dc2_pipeline_bps_step1.yaml]: Remove extra lines/spaces as suggested.

**Review by taranu — CHANGES_REQUESTED** [review]

File: .github/workflows/yamllint.yaml (hunk 1 above)
Comment (brendal4) [reply .github/workflows/yamllint.yaml:8]: Done (I hope I put it in the right place)

File: weekly_templates/dc2_pipeline_bps_step1.yaml (hunk 4 above)
Comment (brendal4) [reply weekly_templates/dc2_pipeline_bps_step1.yaml]: I removed the extra line at the end of each config file, but I left some spaces between sections of the file for clarity and easier reading. Hope this is kosher.

**Comment by brendal4** [comment]
See new changes and comments...

**Review by taranu — APPROVED** [review]

**Pull request merged**